packages/commonwealth/client/scripts/views/pages/user_dashboard/user_dashboard_row_bottom.tsx

```diff
@@ -23,6 +24,22 @@ type UserDashboardRowBottomProps = {
 
 export const UserDashboardRowBottom = (props: UserDashboardRowBottomProps) => {
   const { threadId, commentCount, commentId, chainId, commenters } = props;
+  const [update, setUpdate] = useState<boolean>(false);
```

Comment:
All we need is a local state update to get the latest from `app` state, so this is just a binary toggle that we flip after `subscribeToThread ` is called. 

This probably works elsewhere, so it _might_ be a better pattern for other recent work I did relying on `app` state.

Reply:
Done. Thanks!